indico/modules/events/operations.py

```diff
@@ -51,6 +52,20 @@ def create_event_references(event, data):
         logger.info('Reference "%s" created by %s', reference, session.user)
 
 
+def create_event(category, event_type, data):
+    from MaKaC.conference import Conference
+    conf = Conference.new(category, creator=session.user, title=data.pop('title'), start_dt=data.pop('start_dt'),
+                          end_dt=data.pop('end_dt'), timezone=data.pop('timezone'), event_type=event_type)
+    event = conf.as_event
+    theme = data.pop('theme', None)
+    if theme is not None:
+        layout_settings.set(event, 'timetable_theme', theme)
+    event.populate_from_dict(data)
+    db.session.flush()
+    logger.info('Event %r created in %r by %r ', event, category, session.user)
```

Comment:
What's the point of adding it to the log? You see creator etc in the event management area itself